openproblems/tasks/chromatin_potential/methods/linear_regression.py

```diff
@@ -0,0 +1,177 @@
+import numpy as np
+import pandas as pd
+from scipy.sparse import csr_matrix
+import scanpy as sc
+import subprocess
+from ....tools.decorators import method
+
+
+def _chrom_limit(x, tss_size=2e5):
+    """This function returns upstream and downstream intervals
+    of tss_size for calculating gene score of ATAC-seq
+
+    @tss_size: upstream and downstream limit size
```

Comment:
Docstrings should follow numpy format. https://numpydoc.readthedocs.io/en/latest/format.html

Discussion of parameter `x` should mention that it is a pd.Series containing [start, end, direction] where start and end are ints and direction is {'+', '-'}.

Reply:
I followed the format from numpydoc to change the docstring.